internal/controller/promotions/promoqueues.go

```diff
@@ -68,20 +67,20 @@ func (pqs *promoQueues) initializeQueues(ctx context.Context, promos kargoapi.Pr
 			continue
 		}
 		pq.Push(&promo)
-		logger.WithFields(log.Fields{
-			"promotion": promo.Name,
-			"namespace": promo.Namespace,
-			"stage":     promo.Spec.Stage,
-			"phase":     promo.Status.Phase,
-		}).Debug("pushed Promotion onto Stage-specific Promotion queue")
+		logger.WithValues(
+			"promotion", promo.Name,
+			"namespace", promo.Namespace,
+			"stage", promo.Spec.Stage,
+			"phase", promo.Status.Phase,
+		).V(1).Info("pushed Promotion onto Stage-specific Promotion queue")
 	}
-	if logger.Logger.IsLevelEnabled(log.DebugLevel) {
+	if logger.V(1).Enabled() {
 		for stage, pq := range pqs.pendingPromoQueuesByStage {
-			logger.WithFields(log.Fields{
-				"stage":     stage.Name,
-				"namespace": stage.Namespace,
-				"depth":     pq.Depth(),
-			}).Debug("Stage-specific Promotion queue initialized")
+			logger.WithValues(
+				"stage", stage.Name,
+				"namespace", stage.Namespace,
+				"depth", pq.Depth(),
+			).V(1).Info("Stage-specific Promotion queue initialized")
```

Comment:
I think this is another case where logr gives us a cleaner way of doing things:

```
logger.Info(
  "Stage-specific Promotion queue initialized",
  "stage", stag.name,
  "namespace", stage.Namespace,
  "depth", pq.Depth(),
)
```

Reply:
I think I got all of them here [c4c69fb](https://github.com/akuity/kargo/pull/1791/commits/c4c69fb3b0145755839faabbe93bc0c8f1a81c68)